src/main/java/com/deep/sample/controller/SampleController.java

```diff
@@ -13,7 +13,7 @@
 public class SampleController {
 
     @Autowired
-    private SampleService sampleService;
+    private transient SampleService sampleService;
```

Comment:
Didn't understand this loc

Reply:
So bean needs to be declared as transient if it is not public or  don't have a public accessor.
Reference : https://pmd.sourceforge.io/pmd-5.4.1/pmd-java/rules/java/javabeans.html#BeanMembersShouldSerialize